com.avaloq.tools.ddk.workflow/src/com/avaloq/tools/ddk/workflow/GenerateFormatterTestLanguage.mwe2

```diff
@@ -1,23 +1,25 @@
 /*******************************************************************************
- * Copyright (c) 2016 Avaloq Group AG and others.
+ * Copyright (c) 2016 Avaloq Evolution AG and others.

```

Comment:
this change in copyright is not correct, could you revert it here and in the rest of the PR?

Reply:
Ok.